packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -44,6 +44,9 @@ export async function processOutboundDocumentQueryResps({
     const docQueryStartedAt = patient.data.documentQueryProgress?.startedAt;
     const duration = elapsedTimeFromNow(docQueryStartedAt);
 
+    const docRefsPromises = results.map(toDocumentReference);
+    const docRefs = (await Promise.all(docRefsPromises)).flat();
```

Comment:
I know it was like this before, but `toDocumentReference` doesn't return a Promise, so there's no need for `Promise.all`. Let's remove it?